tests/conftest.py

```diff
@@ -202,3 +205,52 @@ def populated_cockroach_database(cockroach_database_connection: Tuple[str, str])
     handle, _ = cockroach_database_connection
     connection = Connection.connections[handle]
     populate_database(connection, include_comments=True)
+
+
```

Comment:
Moved up to conftest from the test_datasources.py module, because test_sql_magics wants it also.